setup.py

```
@@ -76,10 +76,10 @@
 			raise ValueError(f"[CWE-706] Access to the file {filename} was not expected.") from None
 		with open(f"./{filename}") as f:
 			theResult = f.read()
-	except Exception as err:
+	except Exception as _cause:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)